From 985c196ddbb4c0182df81a5f989c197544f9ff9e Mon Sep 17 00:00:00 2001
From: Gerd Hoffmann <kraxel@redhat.com>
Date: Tue, 26 Sep 2017 10:57:32 +0200
Subject: [PATCH 1/4] qemu-char: add Czech characters to VNC keysyms

RH-Author: Gerd Hoffmann <kraxel@redhat.com>
Message-id: <20170926105734.746-2-kraxel@redhat.com>
Patchwork-id: 76566
O-Subject: [RHEL-7.5 qemu-kvm PATCH 1/3] qemu-char: add Czech characters to VNC keysyms
Bugzilla: 1476641
RH-Acked-by: Laurent Vivier <lvivier@redhat.com>
RH-Acked-by: Markus Armbruster <armbru@redhat.com>
RH-Acked-by: John Snow <jsnow@redhat.com>

From: Jan Krupa <JKrupa@suse.com>

This patch adds missing Czech characters to the VNC keysym table.

Signed-off-by: Jan Krupa <jkrupa@suse.com>
Signed-off-by: Michael Tokarev <mjt@tls.msk.ru>
(cherry picked from commit 018715f96cc1315df355743a1f807e1b95d2a10f)
Signed-off-by: Miroslav Rezanina <mrezanin@redhat.com>
---
 ui/vnc_keysym.h | 8 ++++++++
 1 file changed, 8 insertions(+)

diff --git a/ui/vnc_keysym.h b/ui/vnc_keysym.h
index 6250bec..72c01d1 100644
--- a/ui/vnc_keysym.h
+++ b/ui/vnc_keysym.h
@@ -224,6 +224,14 @@ static const name2keysym_t name2keysym[]={
 { "odoubleacute",         0x1f5},
 { "udoubleacute",         0x1fb},
 
+/* Czech national characters */
+{ "ecaron",               0x1ec},
+{ "scaron",               0x1b9},
+{ "ccaron",               0x1e8},
+{ "rcaron",               0x1f8},
+{ "zcaron",               0x1be},
+{ "uring",                0x1f9},
+
     /* modifiers */
 {"ISO_Level3_Shift", 0xfe03}, /* XK_ISO_Level3_Shift */
 {"Control_L", 0xffe3}, /* XK_Control_L */
-- 
1.8.3.1

